src/hooks/useContract.ts

```diff
@@ -3,14 +3,19 @@ import { JsonRpcSigner, Web3Provider } from '@ethersproject/providers';
 import { useMemo } from 'react';
 import PREMIUM_MEMBERSHIP_CONTRACT_ABI from 'abis/premium-membership-contract.json';
 import GENERAL_MEMBERSHIP_CONTRACT_ABI from 'abis/general-membership-contract.json';
-import GALLERY_MEMORABILIA_CONTRACT_ABI from 'abis/gallery-memorabilia-contract.json';
+import GALLERY_MEMENTOS_CONTRACT_ABI from 'abis/gallery-mementos-contract.json';
 import { network } from 'connectors/index';
 import { useActiveWeb3React } from './useWeb3';
 
-export const PREMIUM_MEMBERSHIP_CONTRACT_ADDRESS = '0xe01569ca9b39E55Bc7C0dFa09F05fa15CB4C7698';
-export const GENERAL_MEMBERSHIP_CONRTACT_ADDRESS = '0xe3d0fe9b7e0b951663267a3ed1e6577f6f79757e';
-
-export const GALLERY_MEMORABILIA_CONTRACT_ADDRESS = '0x35ae1f85389D2bbd48113909DA6182304952f809';
+export const PREMIUM_MEMBERSHIP_CONTRACT_ADDRESS =
+  process.env.NEXT_PUBLIC_PREMIUM_MEMBERSHIP_CONTRACT_ADDRESS ||
+  '0xe01569ca9b39E55Bc7C0dFa09F05fa15CB4C7698';
```

Comment:
Lmk if we wanna remove the fallback contract address

Reply:
but otherwise PR looks good!!

Reply:
+1, thank you jakz!!